### ISSUE TITLE
Add support for unsigned integer to migration schema builder 

### ISSUE THREAD

**Issue by lisi4ok** [yiisoft/yii2#9443]
Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: yii\db\ColumnSchemaBuilder::unsigned()'

in /var/www/ivm/vendor/yiisoft/yii2/base/Object.php:220

My code:

	public function up() {
		$this->createTable('auditLog', [
			'id' => $this->primaryKey(),
			'model' => $this->string(255)->notNull(),
			'action' => $this->string(255)->notNull(),
			'oldAttributes' => $this->text(),
			'newAttributes' => $this->text(),
			'IPAddress' => $this->string(255),
			'createdAt' => $this->datetime(),
			'updatedAt' => $this->datetime(),
			'createdBy' => $this->integer(10)->unsigned(),
			'updatedBy' => $this->integer(10)->unsigned(),
		]);
	}

**Comment by samdark** [comment]
There's no support for it.

**Comment by samdark** [comment]
Yet.

**Comment by bryglen** [comment]
Bump for this. we need this

**Comment by cebe** [comment]
you can just use old syntax to specify and custom type you like. ColumnSchemaBuilder is mostly useful to create schema independed defintions.

**Comment by lisi4ok** [comment]
@cebe  I don`t like old syntax. So i report this error.
Please release the new version more faster. I can`t w8 for the new syntax. Its soo easy :P

**Comment by rezident** [comment]
You can use on version < 2.0.7
```php
$this->createTable('{{%ipgeo}}', [
      'from' => $this->integer() . ' UNSIGNED NOT NULL',
      'to' => $this->integer() . ' UNSIGNED NOT NULL'
]);
```

**Comment by lisi4ok** [comment]
I know but it looks ugly to me.
Better to use only 1 way. Only OLD WAY or Only NEW WAY.
When will be released -v 2.0.7 ?

**Comment by rezident** [comment]
My answer not only for you. I had this problem too. I agree about one way but I admit use OLD WAY temporary. Still I use this code (before -v 2.0.7):
~~~php
$this->createTable('{{%ipgeo}}', [
      'from' => $this->integer() . ' UNSIGNED NOT NULL', //@todo ->notNull()->unsigned(),
      'to' => $this->integer() . ' UNSIGNED NOT NULL', //@todo ->notNull()->unsigned(),
      'vocabulary_id' => $this->integer()->notNull()
]);
~~~
